src/main/scala/com/exasol/cloudetl/parquet/converter/ParquetConverter.scala

```diff
@@ -60,6 +95,23 @@ final case class ParquetStringConverter(index: Int, holder: ValueHolder)
     holder.put(index, decodedDictionary(dictionaryId))
 }
 
+/**
+ * A converter for {@code DECIMAL} annotated Parquet types.
+ *
+ * The decimal annotation can be used for the following Parquet types:
+ * {@code INT32}, {@code INT64}, {@code FIXED_LEN_BYTE_ARRAY} and {@code
+ * BINARY}.
+ *
+ * The following schema fits this converter:
+ * {{{
```

Comment:
That's also scala specific?

Reply:
Yes, exactly. It has it is own renderer. `{{{` can be used for larger code blocks.

An example api doc page:
![Screenshot from 2020-11-30 11-20-12](https://user-images.githubusercontent.com/916295/100599818-c864d500-3300-11eb-80b9-b0e4093f4284.png)